### DISCUSSION TITLE
Touch support for osu!mania and other rulesets

### DISCUSSION THREAD

**Discussion by mobilex1122** [ppy/osu#13274] [Ideas] [answered]
I like using osu on my phone but I can play only normal osu on toutch screen. It will be very good to add support to press osu!mania keys on phone and play another modes like in melody. 

Thanks for reading

(sorry for my English)

**Comment by gagahpangeran** [comment, marked as answer]
Tracked in https://github.com/ppy/osu/issues/4201

**Reply by mobilex1122** [reply]
Ou sorry I missed it.
